reactive/kubernetes_master.py

```diff
@@ -2348,6 +2421,9 @@ def poke_network_unavailable():
     discussion about refactoring the affected code but nothing has happened
     in a while.
     """
+    public_address, public_port = kubernetes_master.get_api_endpoint()
+    public_server = 'https://{0}:{1}'.format(public_address, public_port)
```

Comment:
I would prefer the "local address/server" here - the ingress address of kube-api-endpoint. For the same reason that we prefer it for kube-scheduler, kube-controller-manager, etc. No need to send API traffic over the network to loadbalancers if we don't have to.

Reply:
+1